pyiron_base/storage/has_stored_traits.py

```diff
@@ -6,17 +6,14 @@
 A parent class for managing input to pyiron jobs.
 """
 
-from __future__ import annotations
```

Comment:
From what I can see in this module, there's no problem removing this here. Be careful though, as there are places this maneuver is [doing real work](https://stackoverflow.com/questions/61544854/from-future-import-annotations).